README.md

```diff
@@ -299,6 +299,12 @@ For more information, refer to the [Logging Documentation](https://openvinotoolk
 
 Note: Set your API Key for [Comet.ml](https://www.comet.com/signup?utm_source=anomalib&utm_medium=referral) via `comet_ml.init()` in interactive python or simply run `export COMET_API_KEY=<Your API Key>`
 
+# Community Projects
+## 1. Pipeline training and inference on website
+In this project, I create a pipeline for training and inference (my PR #729) with all datasets format MVTec (custom) on UI. Everyone can training with all Anomalib models supported and quickly to demo. And my project is an end-to-end example in anomaly detection task. 
+
+For more details, everyone can visit and use my project below link. <p><a href="https://github.com/openvinotoolkit/anomalib/discussions/733">Link Show and tell</a> | <a href="https://github.com/vnk8071/anomaly-detection-in-industry-manufacturing/tree/master/anomalib_contribute">Github anomalib vnk8071</a></p>
+
```

Comment:
Minor suggestion for edits.

```suggestion
# Community Projects
## 1. Web-based Pipeline for Training and Inference
This project showcases an end-to-end training and inference pipeline build on top of Anomalib. It provides a web-based UI for uploading MVTec style datasets and training them on the available Anomalib models. It also has sections for calling inference on individual images as well as listing all the images with their predictions in the database.

You can view the project on [Github](https://github.com/vnk8071/anomaly-detection-in-industry-manufacturing/tree/master/anomalib_contribute)
For more details see the [Discussion forum](https://github.com/openvinotoolkit/anomalib/discussions/733)
```

Reply:
@ashwinvaidya17 Thanks for your suggestion and I have already change the commit.